app/frontend/src/components/EditLocationMap.tsx

```diff
@@ -8,45 +18,54 @@ import {
 } from "maplibre-gl";
 import React, { useRef, useState } from "react";
 
-import Alert from "./Alert";
-import Map from "./Map";
-import MapSearch from "./MapSearch";
+import { userLocationMaxRadius, userLocationMinRadius } from "../constants";
 
 const useStyles = makeStyles({
+  root: {
+    margin: "auto",
+    maxWidth: 700,
+  },
+  map: {
+    height: 400,
+    position: "relative",
+  },
   grow: {
     height: "100%",
     width: "100%",
   },
-  root: {
-    height: 200,
-    position: "relative",
-    width: 400,
+  coordinateBox: {
+    textAlign: "right",
+  },
+  displayLocation: {
+    width: "100%",
   },
 });
 
-export interface Location {
+export interface ApproximateLocation {
+  address: string;
   lat: number;
   lng: number;
+  radius: number;
 }
 
 export interface EditLocationMapProps extends BoxProps {
-  // text
-  address?: string;
-  // lat+lng
-  location?: Location;
-  setAddress: (value: string) => void;
-  //this function is called on mouse release
-  setLocation: (value: Location) => void;
+  location?: ApproximateLocation;
+  // this function is called on mouse release
+  setLocation: (value: ApproximateLocation) => void;
   grow?: boolean;
+  // whether to hide the radius slider
+  hideRadiusSlider?: boolean;
+  // whether we are selecting an exact point (for pages, etc) or approx circle, doesn't maeks ense with radius slider
+  exact?: boolean;
 }
 
 export default function EditLocationMap({
-  address,
   location,
-  setAddress,
   setLocation,
   className,
   grow,
+  hideRadiusSlider,
```

Comment:
default to true

Reply:
changed to `showRadiusSlider` defaulted to false